SetupDataPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md

```diff
@@ -149,9 +153,118 @@ will omit the integration steps for these features. For more information about D
   INF SetupDataPkg/ConfApp/ConfApp.inf
 ```
 
-1. Remove the FDF sections below.
+Remove the FDF sections below.
 
 ``` bash
 [FV.YOUR_DXE_FV]
   # INF MdeModulePkg/Application/UiApp/UiApp.inf
 ```
+
+## Profiles Integration
+
+In order to use configuration profiles, the platform must include the above changes as well as include a YAML file that
+contains the default values for the generic profile. If additional profiles are required, the platform must include YAML
+delta files (.dlt) for each profile that are overrides on top of the generic profile.

```

Comment:
should mention how the .dlt files are created here through ConfigEditor "Save Config Changes to Change File" vs "Save Full Config Changes to Change FIle".

.dlt is doens't need to be mentioned, as it is necessary to be named that.

Reply:
Thanks, will add.